utils/sysfs/sysfs_test.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2020 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sysfs
+
+import (
+	"github.com/stretchr/testify/assert"
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+)
+
+func TestGetNodes(t *testing.T) {
+	//overwrite global variable
```

Comment:
This variable is not really global as it is not exported. Can you change "gloabl" to "package" in all comments? Global state is evil.